pkg/resources/schema.go

```diff
@@ -128,6 +131,7 @@ func Schema() *schema.Resource {
 
 func ImportSchema(ctx context.Context, d *schema.ResourceData, meta any) ([]*schema.ResourceData, error) {
 	log.Printf("[DEBUG] Starting schema import")
+	ctx = tracking.NewContext(ctx, tracking.NewVersionedMetadata(resources.Schema, tracking.ImportOperation))
```

Comment:
Why is this needed here and not in other functions?

Reply:
Seems to be leftover from previous approach, removed